docs/operators/to_sentinelone_data_lake.md

```diff
@@ -0,0 +1,64 @@
+---
+title: to_sentinelone_data_lake
+category: Outputs/Events
+example: 'to_sentinelone_data_lake "https://…", …'
+---
+
+Sends security events to SentinelOne Singularity Data Lake via REST API.
+
+```tql
+to_sentinelone_data_lake url:string, bearer_token=string
+                        [session_info=record, timeout=duration]
+```
+
+## Description
+
+The `to_sentinelone_data_lake` operator sends incoming events to
+the [SentinelOne Data Lake REST API](https://support.sentinelone.com/hc/en-us/articles/360004195934-SentinelOne-API-Guide)
+as structured data.
+
+The operator accumulates multiple events before sending them as a single batch
+request, respecting the API's limits.
+
+If events are OCSF events, the timestamp and severity are automatically extracted
+and added to the events meta information.
+
+### `url: string`
+
+The URL of the SentinelOne Data Lake's console, for example `https://xdr.us1.sentinelone.net`
+
+### `bearer_token = string`
+
+The _token_ to use for authorization.
+
+### `session_info = record (optional)`
+
+Some additional sessionInfo to send with each batch of events, as the
+`sessionInfo` field in the request body.
+
+### `timeout = duration (optional)`
+
+The delay after which events are sent, even if if this results in fewer events
+sent per message.
+
+Defaults to `5min`.
+
+## Automatic OCSF detection
```

Comment:
I would put this section simply at the bottom of `## Description` and not create a separate section.